File: mario.c
#include <stdio.h>
#include <cs50.h>


void print_p (int length);

int main (void)
{
    int height;
    // User introduce a postive integer
    do
    {
        height = get_int ("Enter the size of the pyramid: ");
    }
    while (height  < 1 || height  > 8);

    // Call the fuction print_p
    print_p (height);

}

//Fuction that prints # for a given number and blank space
void print_p (int length)
{
    for (int i = 0; i < length; i ++)
    {

        for (int j = 0; j < length; j ++)
        {
            if (i + j < length - 1)
            {
                printf (" ");
            }
            else
            {
                printf ("#");
            }
        }
        printf ("\n");
    }
}

File: mario.py
# A simple program to draw a pyramid between 1-8 blocks reciving the height from the user
def main():
    height = 0

    while height <= 0 or height > 8:
        try:
            height = int(input("Enter height of the pyramid between 1-8: "))

        except:
            print("Enter an integer")

    draw(height)


def draw(height):

    for i in range(height):
        for j in range(height):
            if i + j < height - 1:
                print(" ", end="")
            else:
                print("#", end="")
        print("")


if __name__ == "__main__":
    main()